pkg/printers/internalversion/describe.go

```diff
@@ -2870,42 +2870,55 @@ func describeHorizontalPodAutoscaler(hpa *autoscaling.HorizontalPodAutoscaler, e
 
 func describeNodeResource(nodeNonTerminatedPodsList *api.PodList, node *api.Node, w PrefixWriter) error {
 	w.Write(LEVEL_0, "Non-terminated Pods:\t(%d in total)\n", len(nodeNonTerminatedPodsList.Items))
-	w.Write(LEVEL_1, "Namespace\tName\t\tCPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\n")
-	w.Write(LEVEL_1, "---------\t----\t\t------------\t----------\t---------------\t-------------\n")
+	w.Write(LEVEL_1, "Namespace\tName\t\tCPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\tGPU Requests\tGPU Limits\n")
+	w.Write(LEVEL_1, "---------\t----\t\t------------\t----------\t---------------\t-------------\t------------\t----------\n")
```

Comment:
This is not gonna be extensible considering that there are many resource (types) to be shown.
Please check if the approach taken by the following patch is more appropriate.

xref: https://github.com/kubernetes/kubernetes/pull/46079

Reply:
I didn't notice this pr before,  but seems to be a pretty good approach!  
Wish it could be released soon. : )